src/Symfony/Component/HttpKernel/MicroKernel.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel;
+
+use Symfony\Component\Config\Loader\Loader;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\Loader\ContainerBuilderAwareLoader;
+
+/**
+ * A Kernel that allows you to configure services.
+ *
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ */
+abstract class MicroKernel extends Kernel
+{
+    /**
+     * Configure dependency injection extensions that have been added to the container.
+     *
+     * $c->loadFromExtension('framework', array(
+     *     'secret' => '%secret%'
+     * ));
+     *
+     * @param ContainerBuilder $c
+     * @param LoaderInterface  $loader
+     */
+    abstract protected function configureExtensions(ContainerBuilder $c, LoaderInterface $loader);
```

Comment:
IMO, it would be better to be more consistent on naming as method signature is always `Container $container` everywhere else in the code